Add Grace's DevBlog post (#2461)
fishtown-analytics/docs.getdbt.com: website/blog/2022-11-30-dbt-project-evaluator.md

---
title: "Introducing the dbt_project_evaluator: Automatically evaluate your dbt project for alignment with best practices "

> description: "The dbt_project_evaluator is a dbt package created by Grace Goheen and other dbt Labs team members to help analytics engineers automatically audit their dbt projects for bad practices. Goodbye auditing nightmares, hello beautiful DAG."

Reviewer: ```suggestion
description: "The dbt_project_evaluator is a dbt package created by the dbt Labs Professional Services team to help analytics engineers automatically audit their dbt projects for bad practices. Goodbye auditing nightmares, hello beautiful DAG."
```
can't take all the credit!!

slug: align-with-dbt-project-evaluator

authors: [grace_goheen]

tags: [analytics craft]
hide_table_of_contents: false

date: 2022-11-30
is_featured: true
---

> ## Why we built this: A brief history of the professional services team at dbt Labs

Reviewer: ```suggestion
## Why we built this: A brief history of the dbt Labs Professional Services team
```
